cmd/up/up.go

```diff
@@ -181,6 +181,8 @@ func Up() *cobra.Command {
 				log.Infof("error deleting deprecated volume: %v", err)
 			}
 
+			err = fmt.Errorf("%w\n    Find additional logs at: %s/okteto.log", err, config.GetOktetoHome())
```

Comment:
Kept 4 spaces here since the icon takes 3 and 1 extra before beginning the sentence.

Reply:
Also should I switch to `log.Information` or would this be fine?